frontend/src/resources/slugs.js

```diff
@@ -20,6 +20,7 @@ const slugs = {
     categories: '/categories',
     inbox: './inbox',
     settings: '/settings',
+    logout: '/logout',
```

Comment:
Might be irrelevant after merging the bug branch.

Reply:
Lets not consider the changes in #78 regarding the logout. We tried it with @manifau yesterday but in the other branch you solved it.